doc/com/financialforce/orizuru/AbstractConsumer.html

```diff
@@ -384,6 +384,6 @@ <h4>getQueueName</h4>
 <!--   -->
 </a></div>
 <!-- ======== END OF BOTTOM NAVBAR ======= -->
-<p class="legalCopy"><small>Copyright &#169; 2017. All rights reserved.</small></p>
```

Comment:
Minor: Should this be 2017-2018?

Reply:
It's auto generated so I'm sticking with the default.